src/structures/typedefs.h

```diff
@@ -16,6 +16,20 @@ All rights reserved (see LICENSE).
 #include <string>
 #include <unordered_set>
 #include <vector>
+#include <optional>
+
+#ifdef _MSC_VER
+  // include support for "and"/"or"
+  #include <iso646.h>
+  // some include defines MAX_PRIORITY..
+  #ifdef MAX_PRIORITY
+    #undef MAX_PRIORITY
+  #endif
+  // ERROR is already defined in WIN
+  #ifdef ERROR
+    #undef ERROR
```

Comment:
I don't get why this is required, since our own `MAX_PRIORITY` and `ERROR` are defined within the `vroom` namespace?

Reply:
Actually I think I can get rid of the ERROR thing at least by providing the `NOGDI` directive during compilation, WinGDI.h defines ERROR as macro. Why that interferes with scoped names... :shrug: No idea what defines `MAX_PRIORITY`, must be some hidden macro as well..

Reply:
NOGDI was fixing both!